navigation/app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/MockNavigationActivity.java

```diff
@@ -191,6 +191,30 @@ public void onFailure(Call<DirectionsResponse> call, Throwable throwable) {
    */
 
   @Override
+  public void onMilestoneEvent(RouteProgress routeProgress, String instruction, int identifier) {
+    Timber.d("Milestone Event Occurred with id: %d", identifier);
+    switch (identifier) {
+      case NavigationConstants.URGENT_MILESTONE:
+        Toast.makeText(this, "High Milestone", Toast.LENGTH_LONG).show();
+        break;
+      case NavigationConstants.IMMINENT_MILESTONE:
+        Toast.makeText(this, "Medium Milestone", Toast.LENGTH_LONG).show();
+        break;
+      case NavigationConstants.NEW_STEP_MILESTONE:
+        Toast.makeText(this, "New Step", Toast.LENGTH_LONG).show();
+        break;
+      case NavigationConstants.DEPARTURE_MILESTONE:
+        Toast.makeText(this, "Depart", Toast.LENGTH_LONG).show();
+        break;
+      case NavigationConstants.ARRIVAL_MILESTONE:
+        Toast.makeText(this, "Arrival", Toast.LENGTH_LONG).show();
+        break;
+      default:
```

Comment:
Added the toast on [line 213](https://github.com/mapbox/mapbox-navigation-android/pull/84/files#diff-9bb31ba6febea31b9cf16d6419899633R213), Github's just not showing it in the snippet above.